vignettes/articles/cookbook_running_the_analysis.Rmd

```diff
@@ -93,13 +93,13 @@ The next step after loading all required input files is to run the matching proc
 
 The matching process is divided into three main steps:
 
-- Apply Matching Algorithm – Calculate match precision scores between the loan book and ABCD datasets.
-- Validate Matches – Manually review matches, selecting the correct company when multiple exceed the threshold score.
-- Prioritize Matches – Rank validated matches by priority level.
+1. Apply Matching Algorithm – Calculate match precision scores between the loan book and ABCD datasets.
+2. Validate Matches – Manually review matches, selecting the correct company when multiple exceed the threshold score.
+3. Prioritize Matches – Rank validated matches by priority level.
```

Comment:
```suggestion
3. Prioritize Matches – Pick validated matches by priority level.
```
I wonder if we should be more precise here. `prioritize()` does not simply rank matches, it picks a match according to the ranking. the other ones are dropped